docs/terminal/basics.md

```diff
@@ -457,6 +457,15 @@ To workaround this you have two options:
 * Ensure that you don't set `$EDITOR` to `vi(m)`. However, this isn't an option if you want your Git editor to work.
 * Add `bindkey -e` to your init script to set Emacs explicitly.
 
+How can I configure a keybinding which maps `Cmd+C` to `Ctrl+C` in order to align with behavior in the macOS terminal?
```

Comment:
Shouldn't this be JSON to put in keybinding.json for an end user?

Reply:
This looks like the VS Code TypeScript code from src\vs\workbench\contrib\terminal\browser\terminal.contribution.ts.
Shouldn't this be a JSON snippet that an end user can add to their keybinding.json?